indico/modules/events/registration/util.py

```diff
@@ -1025,7 +1025,10 @@ def process_registration_picture(source, *, thumbnail=False):
     size_x, size_y = picture.size
     if max(size_x, size_y) > max_size:
         ratio = max_size / max(size_x, size_y)
-        picture = picture.resize((int(ratio * size_x), int(ratio * size_y)), Image.Resampling.BICUBIC)
+        try:
+            picture = picture.resize((int(ratio * size_x), int(ratio * size_y)), Image.Resampling.BICUBIC)
```

Comment:
Is size_x or size_y zero here or does it become zero because it's 1 and then gets multiplied with something less than 1?

If that's the case, `max(1, int(...))` would avoid this w/o having to catch a rather broad exception here.

Reply:
Yes, you're right it is because the the value is small and gets multiplied by the ratio and causes it to be 0 after casting to int. 

But at the same time if we do not raise an exception here then it means images with width/height of 1 would be accepted and saved without any issues?

Reply:
Let's actively reject such - indeed clearly nonsensical - sizes then - better than catching any ValueError and silencing it.

Reply:
This is an image appearing as just a line: <img width="691" alt="Screenshot 2024-10-31 at 16 32 25" src="https://github.com/user-attachments/assets/bb8b16da-9966-413b-ac42-abf3c9f36243">

This is another image when `minCropSize={minPictureSize || 25}` is being used so I couldn't crop to a line. But it's still not a good image
<img width="707" alt="Screenshot 2024-10-31 at 16 33 23" src="https://github.com/user-attachments/assets/eff64f78-26e1-4077-b621-2b39861efc52">

Reply:
> Let's actively reject such - indeed clearly nonsensical - sizes then - better than catching any ValueError and silencing it.

Would checking the ratio be a good way to do this? Because for example a user could upload a `120 * 100` image and it could contain a face, but if it is `1200 * 25` then it's clearly not a good image

Reply:
I think it's hard to determine what makes sense and what doesn't... it could even depends on what such a field is used for.

Of course anything with single-digit sizes counts is probably garbage, but how likely is it that someone actually sends this with a real registration. And in the end it's up to the event organizer to do something with the pictures in most cases (after all, that's why they ask their participants for them!), so they'll know what to do with useless ones (e.g. rejecting/deleting the registration, or asking the person to provide something better).

Reply:
Right, it's very unlikely. In fact we only got this error once.